paper/src/main/java/vg/civcraft/mc/civmodcore/world/locations/chunkmeta/ChunkCoord.java

```diff
@@ -190,6 +190,18 @@ void minecraftChunkLoaded() {
 			}
 		}
 	}
+	
+	public boolean isChunkLoaded() {
+		if (this.lastUnloadingTime <= 0) {
+			//being loaded rn, ignore
+			return false;
+		}
+		if (this.lastUnloadingTime > 0) {
```

Comment:
Unnecessary if-statement. Can be simplified to:
```java
public boolean isChunkLoaded() {
    return this.lastUnloadingTime > 0 // being loaded rn, ignore
            && this.lastUnloadingTime < this.lastLoadingTime;
}
```

Reply:
I thought about something like that when writing it, but decided on the longer version for the sake of readability.